modules/spo-configuring-webhooks.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * security/security_profiles_operator/spo-advanced.adoc
+
+:_content-type: PROCEDURE
+[id="spo-configuring-webhooks_{context}"]
+= Configuring webhooks
+
+Both profile binding and profile recording make use of webhooks. Their configuration (an instance of `MutatingWebhookConfiguration` CR) is managed by SPO itself and is not part of the deployed YAML manifests. While the defaults should be acceptable for the majority of users and the webhooks do nothing unless an instance of either `ProfileBinding` or `ProfileRecording` exists in a namespace, it might still be useful to configure the webhooks.
+
+To change the configuration of the webhook, the `spod` CR exposes a `webhookOptions` object that allows the `failurePolicy` and the `namespaceSelector` variables to be set. This allows you to set the webhooks to "soft-fail" or restrict them to a subset of a namespaces so that even if the webhooks failed, other namespaces or resources are not affected.
```

Comment:
a total nit, but here we say that `webhookOptions` allow you to configure `failurePolicy` and `namespaceSelector` but below we use `objectSelector` as an example. Could we mention the `objectSelector` here as well?

Reply:
As in, list all three? 

"... to configure `failurePolicy`, `namespaceSelector`, and `objectSelector` variables to be set." ?

Reply:
yes, or we could say that the user can `oc explain spod.spec.webhookOptions` and below is an example of setting `objectSelector`.